client.go

```diff
@@ -119,7 +127,7 @@ func clientListenToMaster() error {
 }
 
 func clientTalkToServer() error {
```

Comment:
Why do we need this function? Client talking to server is not a separate thread and should happen according to the corresponding commands from the master.

Reply:
I can delete it, it was a part of the wrapper code :') (y)